cime_config/namelist_definition_blom.xml

```diff
@@ -903,6 +892,26 @@
     <desc>0 = netcdf, 1 = pnetcdf</desc>
   </entry>
 
+  <entry id="use_diag">
+    <type>logical</type>
+    <category>limits</category>
+    <group>limits</group>
+    <values>
+      <value>.false.</value>
+    </values>
+    <desc>optionally turn on additional diagnostics</desc>
+  </entry>
+
+  <entry id="use_arctic">
+    <type>logical</type>
+    <category>limits</category>
+    <group>limits</group>
+    <values>
+      <value>.true.</value>
+    </values>
+    <desc>if region includes arctic ocean</desc>
+  </entry>
+
```

Comment:
Question out of curiosity: what is the use_arctic switch for?

Reply:
use_arctic has a different communication pattern in mod_xc.F90. 
In particular - it checks that all arctic patch tiles must be the same size or empty,
and empty tiles must be "twinned" across the top boundary. It ensures that the latitudinal tile dimension is closed/arctic and handles the halo in the arctic differently. @matsbn - do you want to add more?